other/edge-server/backend/data_sources/mongodb-atlas/todo/Item/rules.json

```
@@ -6,7 +6,7 @@
             "name": "readAndWriteAll",
```

I suggest renaming this so we make it clearer this is the role that applies to the Edge Server. I know people can infer it from the `apply_when` expression, but it doesn't hurt to make it more explicit.

```suggestion
            "name": "edgeServerReadWriteAll",
```